src/content/hinting.ts

```diff
@@ -749,8 +748,7 @@ function pushSpace() {
 export function hintables(selectors = DOM.HINTTAGS_selectors, withjs = false) {
     let elems = DOM.getElemsBySelector(selectors, [])
     if (withjs) {
-        elems = elems.concat(DOM.hintworthy_js_elems)
-        elems = unique(elems)
+        elems = [...elems, ...DOM.hintworthy_js_elems]
```

Comment:
Aren't we going to get duplicates if there are elements that exist both in `elems` and in `hintworthy_js_elems`?

Reply:
Well, I was just cope with the original code. We can stuff all the elems into Set to get a unique values.

Reply:
It's just shows that using Set to ensure unique is much faster than iterating lambdas in javascript. :) 

BTW, that test page has 0 javascript hintables.